lib/ElementHandle.js

```diff
@@ -152,6 +152,37 @@ class ElementHandle extends JSHandle {
     const clip = Object.assign({}, boundingBox);
     clip.x += pageX;
     clip.y += pageY;
+
+    if (options.padding) {
+      if (options.padding.constructor !== Array || options.padding.length > 4)
+          throw Error('Invalid padding parameter (should be [top, right, bottom, left])');     
+      switch (options.padding.length){
+        case 1:
+          clip.x -= options.padding[0];
+          clip.y -= options.padding[0];
+          clip.width += options.padding[0];
+          clip.height += options.padding[0];
```

Comment:
Shouldn't this be:
`clip.height += options.padding[0] + options.padding[0]`?
A 2x2 square with a padding of 3 should be an 8x8 square.